content/topics/solo-learn/python/9-regular-expressions/_index.md

```diff
@@ -1,16 +1,16 @@
 ---
-content_type: project
+content_type: topic
```

Comment:
These have not been allocated a dbid when consumed by tilde, not sure if there is a problem?

Reply:
I'll look into it